src/protocol/protocol.ts

```diff
@@ -787,7 +787,18 @@ export namespace Script {
     Script.MessageParameters
   >;
 
+  export type RealmCreatedEvent = EventResponse<
+    EventNames.RealmCreated,
+    RealmInfo
+  >;
+
+  export type RealmCreated = {
+    method: 'script.realmCreated';
+    params: RealmInfo;
+  };
+
   export enum EventNames {
+    RealmCreated = 'script.realmCreated',
     MessageEvent = 'script.message',
```

Comment:
Please sort the event names

Could also wrap `// keep-sorted` around here, but not absolutely needed as there are only two entries